pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/omstillingsstoenad/aktivitetsplikt/OmstillingsstoenadAktivitetspliktInformasjon6mndInnhold.kt

```diff
@@ -48,7 +48,7 @@ object OmstillingsstoenadAktivitetspliktInformasjon6mndInnhold : EtterlatteTempl
             outline {
                 paragraph {
                     text(
-                        Bokmal to "Du har omstillingsstønad.",
+                        Bokmal to "Du mottar omstillingsstønad.",
                         Nynorsk to "Du har omstillingsstønad.",
```

Comment:
Burde det vært en oversettelse for nynorsk også?

Reply:
jepp, la det til på det andre brevet også etter samtale med Mari